app/faceanalysis/models/database_manager.py

```diff
@@ -1,39 +1,26 @@
-import os
+from functools import lru_cache
 from sqlalchemy.orm import sessionmaker
 from sqlalchemy import create_engine
 from ..log import get_logger
+from ..settings import (MYSQL_USER, MYSQL_PASSWORD, MYSQL_DATABASE,
```

Comment:
In previous PRs, ..imports were replaced with module.module.module. Which should we choose for consistency?

Reply:
I'll fix up all the imports in a follow up PR, keeping local consistency for now. In general, absolute imports are preferred since they are more explicit.